packages/core/src/components/carousel/carousel.tsx

```diff
@@ -1,142 +1,198 @@
+    this.handleSwipe()
+  }
+
+  handleSwipe = () => {
+    const lastIndex = this.carouselItems.length - 1
+    const isSwipeLeft = this.touchEndX < this.touchStartX
+
+    this.currentIndex = isSwipeLeft
+      ? Math.min(this.currentIndex + 1, lastIndex)
+      : Math.max(this.currentIndex - 1, 0)
+
+    this.showOrHideNavigationButtons()
+  }
+
+  render() {
     return (
       <Host>
-        <swiper-container
-          class='atom-carousel'
-          navigation={this.navigation}
-          space-between={this.spaceBetween}
-          loop={this.loop}
-          speed={this.speed}
-          slides-per-group={this.slidesPerGroup}
-          slides-per-view={this.slidesPerView}
-          free-mode={this.freeMode}
-          autoplay={this.autoplay}
-          autoplay-delay={this.autoplayDelay}
-          injectStyles={injectStyles}
-          touch-start-prevent-default='false'
-          center-insufficient-slides={this.centerInsufficientSlides}
-          centered-slides={this.centeredSlides}
-          watch-slides-progress={this.watchSlidesProgress}
-        ></swiper-container>
+        <div class='atom-carousel' role='region' aria-label='Carousel'>
+          {this.hasNavigation && (
+            <button
+              class='carousel-navigation navigation--prev'
+              role='button'
+              aria-label='Previous'
+              onClick={(event) => this.handleNavigationClick(event)}
+            >
+              <atom-icon icon='chevron-left'></atom-icon>
+            </button>
+          )}
+          <div class='carousel-sliders'>
+            <div class='carousel-wrapper' role='list'>
+              <slot />
+            </div>
+          </div>
+          {this.hasNavigation && (
+            <button
+              class='carousel-navigation navigation--next'
+              role='button'
+              aria-label='Next'
+              onClick={(event) => this.handleNavigationClick(event)}
+            >
+              <atom-icon icon='chevron-right'></atom-icon>
+            </button>
+          )}
+        </div>
+        {this.hasPagination && (
+          <div
+            class='carousel-pagination'
+            role='navigation'
+            aria-label='Pagination'
+          >
+            {Array.from(this.carouselItems).map((_, index) => (
+              <button
+                class={`carousel-pagination__item ${index === this.currentIdx ? 'active' : ''} ${this.thumbnails?.length > 0 ? 'carousel-pagination--thumbnails' : 'carousel-pagination--bullets'}`}
+                role='tab'
+                aria-selected={index === this.currentIdx}
+                aria-controls={`carousel-item-${index}`}
+                onClick={() => this.handleMoveToPaginationItem(index)}
+              >
+                {this.thumbnails?.length > 0 ? (
+                  <img
+                    class='thumbnail'
+                    src={this.thumbnails[index]}
+                    alt={`${index + 1}`}
```

Comment:
(question): Não podemos passar somente o valor do `index + 1`?

```suggestion
                    alt={index + 1}
```

Reply:
`alt` need to be a string